src/main/scala/uk/gov/hmrc/ct/accounts/frs10x/abridged/AC5058A.scala

```diff
@@ -20,6 +20,11 @@ import uk.gov.hmrc.ct.accounts.frs10x.abridged.retriever.AbridgedAccountsBoxRetr
 import uk.gov.hmrc.ct.box._
 
 case class AC5058A(value: Option[String]) extends CtBoxIdentifier(name = "Balance sheet - Creditors within 1 year note.") with CtOptionalString with Input with ValidatableBox[AbridgedAccountsBoxRetriever] {
-  override def validate(boxRetriever: AbridgedAccountsBoxRetriever): Set[CtValidation] =
-    validateOptionalStringByLength("AC5058A", this, 0, 20000) ++ validateOptionalStringByRegex("AC5058A", this, validCoHoCharacters)
+  override def validate(boxRetriever: AbridgedAccountsBoxRetriever): Set[CtValidation] = {
```

Comment:
fyi
after my pull request is merged, we are going to switch to a new validation api. So this style will be deprecated